tests/test_file_storage.py

```diff
@@ -0,0 +1,66 @@
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webelement import WebElement
+
+MAIN_PAGE = "http://localhost:3000/filestorage"
+
+
+def test_upload_file(driver: WebElement):
+    driver.get(MAIN_PAGE)
+    email = "flapmachinist@gmail.com"
+    password = "hello123"
+    email_box = driver.find_element(
+        By.XPATH,
+        "/html/body/div[1]/div/div/div/div/div[2]/div/div/form/div/fieldset/div[1]/div/div/input",
+    )
+    email_box.send_keys(Keys.TAB)
+    email_box.clear()
+    email_box.send_keys(email)
+
+    password_box = driver.find_element(
+        By.XPATH,
+        "/html/body/div[1]/div/div/div/div/div[2]/div/div/form/div/fieldset/div[2]/div/div[1]/input",
+    )
+    password_box.send_keys(Keys.TAB)
+    password_box.clear()
+    password_box.send_keys(password)
+
+    driver.find_element(
+        By.XPATH,
+        "/html/body/div[1]/div/div/div/div/div[2]/div/div/form/div/button",
+    ).click()
```

Comment:
@theNafsters  i think you can refactor this part of the code because almost the same code is being used inside the test_delete_file so maybe you can create a function and when you need to use the code just call the function. right now the line duplication is to high on sonarclould

Reply:
also since you are using same email and password for all test make them global

Reply:
also the Credentials should be stored outside of the code in a configuration file,